### PR TITLE
Adding check to ensure build-all.sh is executed from the libs directo…

### PR DESCRIPTION
**Pull request by hardik25**
Submitting patch for review. Added a check to ensure current working directory is libs.

### CODE REVIEW

File: libs/build-all.sh [hunk 1]
```diff
@@ -15,6 +15,12 @@ NSS_SHA256="fb2d54d507ceb185bac73f492cce7086a462d41977c2378aba9dd10e04448cf3"
 
 # End of configuration.
 
+if [ "${PWD##*/}" != "libs" ]
```
Comment (eoger) [inline libs/build-all.sh]: IIUC this means the script can only be executed if the cwd is a folder named `libs`. Alternatively you could have done `if [ ! -f "$(pwd)/build-all.sh" ];`

File: libs/build-all.sh [hunk 2]
```diff
@@ -15,6 +15,12 @@ NSS_SHA256="fb2d54d507ceb185bac73f492cce7086a462d41977c2378aba9dd10e04448cf3"
...
+then
+    echo "build-all.sh must execute from within libs dir"
```
Comment (eoger) [inline libs/build-all.sh]:
```suggestion
    echo "build-all.sh must be executed from within the libs/ directory."
```

**Comment by eoger** [comment]
Could you sign your commit? It's blocking landing.

**Comment by hardik25** [comment]
@eoger, I tried signing the commit. My last commit shows "Verified" and signed using GPG key. It's still not going through, any ideas why?

**Comment by eoger** [comment]
This is because your first commits are unsigned, I would recommend squashing them in a single signed commit.

**Pull request closed without merging**